src/platform/datapath_raw_xdp_win.c

```diff
@@ -24,8 +24,9 @@
 #endif
 
 typedef struct XDP_DATAPATH {
-    CXPLAT_DATAPATH;
     DECLSPEC_CACHEALIGN
+    CXPLAT_DATAPATH_RAW;
```

Comment:
Why did you move this to after the alignment?

Reply:
not needed..... I changed experimentally....